apps/server/src/modules/authorization/README.md

```diff
@@ -172,12 +176,13 @@ The authorisation module is the core of authorisation. It collects all needed in
 
 ### Reference.loader
 
-For situations where only the id and the domain object (string) type is known, it is possible to use the *ByReferences methods.
+For situations where only the id and the domain object (string) type is known, it is possible to use the \*ByReferences methods.
 They load the reference directly.
+
 > Please keep in mind that it can have an impact on the performance if you use it wrongly.
 > We keep it as a seperate method to avoid the usage in areas where the domain object should exist, because we see the risk that a developer could be tempted by the ease of only passing the id.
 
-### shared/domain/rules/*
+### shared/domain/rules/\*
```

Comment:
This is not a js regex only a path description. I think shared/domain/rules/* is more readable.

Reply:
This was auto formatting from prettier. I enabled it for all files lately because of some discussion over formatting of yaml-files with Sergej and forgot about it. It is escaping the asterisk as a special character in Markdown. In the rendered version it shows up correctly. Shall I revert the changes?